plugins/amazonq/codewhisperer/jetbrains-community/tstFixtures/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererTestUtil.kt

```diff
@@ -69,6 +69,10 @@ object CodeWhispererTestUtil {
     const val codeWhispererCodeScanActionId = "codewhisperer.toolbar.security.scan"
     const val testValidAccessToken = "test_valid_access_token"
     const val testNextToken = "test_next_token"
+    const val qInlineAcceptActionId = "codewhisperer.inline.accept"
+    const val qInlineNavigatePrevActionId = "codewhisperer.inline.navigate.previous"
+    const val qInlineNavigateNextActionId = "codewhisperer.inline.navigate.next"
+    const val qInlineForceAcceptActionId = "codewhisperer.inline.force.accept"
```

Comment:
i feel they could live in `CodeWhispererConstants.kt` as we're using them in src as well https://github.com/aws/aws-toolkit-jetbrains/pull/5270/files#diff-c2086b8e15ec25c5e5aacb3f5f7e070b20c1c44bef8d4deffc01ee5575bc2058R54
```
object ActionId {
    const val qInlineAcceptActionId = ""
    ...
}
```

Reply:
sure sounds good.